config/customization.rb.template

```diff
@@ -0,0 +1,2 @@
+ADMIN_EMAIL = "betydb@gmail.com"
```

Comment:
> settings in the two files seem largely orthogonal

depends on if you are a splitter vs. a lumper. Currently `additional_environment.rb` has information about how to send emails and this `customization.rb` file has information about where to send the emails. So that doesn't seem orthogonal. More generally,

>  if I'm going to eventually jettison this infrastructure in favor of something more comprehensive,

I think that is what needs to be done. We can have a single file and instructions for each line documented, similar to what we do with [the `pecan.xml` settings file](https://github.com/PecanProject/pecan/wiki/PEcAn-Configuration). A more relevant example would be the way Jekyll uses a [`_config.yml`](https://mmistakes.github.io/so-simple-theme/theme-setup/#site-setup) file.

> A less good reason was that I wasn't that familiar with additional_environment.rb and wanted to start somewhere fresh. 

Luckily we have the author Rob here ...